edgelet/iotedge/src/config/apply.rs

```diff
@@ -183,6 +185,7 @@ fn execute_inner(
         manifest_trust_bundle_cert,
         aziot,
         agent,
+        product_info,
```

Comment:
Nit: can we call this host_info? Product_info is the field on the SDK where we put it, but it is really host_info..